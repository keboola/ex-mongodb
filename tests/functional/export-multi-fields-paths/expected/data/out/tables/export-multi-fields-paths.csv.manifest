{"incremental":false,"columns":["id","name"],"column_metadata":{"id":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"name":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}]}}