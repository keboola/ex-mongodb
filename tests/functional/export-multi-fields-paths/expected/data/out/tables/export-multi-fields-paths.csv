"id","name"
"5716054bee6e764c94fa8c8b","Nb. National Bakery"
"5716054cee6e764c94faba0d","La Rosa Bakery"
"5716054cee6e764c94fad056","Emilio Super Bakery Corp"